ui/components/connections/index.js

```diff
@@ -248,9 +248,7 @@ function Connections({
                   ping(tableMeta.rowData[3], tableMeta.rowData[2], tableMeta.rowData[0]);
                 }
               }}
-              iconSrc={
-                getColumnValue(tableMeta.rowData, 'kindLogo', columns)?.colorIcon.split('public')[1]
-              }
+              iconSrc={getColumnValue(tableMeta.rowData, 'kindLogo', columns)?.colorIcon}
```

Comment:
colorIcon??

Reply:
@MUzairS15 Added icon according to the color mode.